presentations/project_update.md

```diff
@@ -0,0 +1,39 @@
+# Project Update Template
+
+## Overview - 1 minute
+
+* Basic background on the company and project
+* What is their business model and what are we doing for them
+
+## Business Goals - 1 minute
+
+* What were the specific business goals of this project?
+* What problem were we solving? What workflow is enabled?
+
+## Uniques - 8 minutes
+### Tech solutions
+
+* What unique technical problems or solutions did we create in the project?
+* What technical solutions might be applicable to other projects?
+* Any new tools or libraries that we would want to use again?
+* Any refinement of existing tools or patterns that is noteworthy?
+
+### Design solutions
+
+* What unique UI/UX problems did we need to address?
+* What design solutions is the team most proud of?
+* What design solutions might be applicable to other projects?
+
+
+### What might be relevant elsewhere
+
+* Advanced Search functionality
+* Notifications, specifically revalidating notifications. Nothing crazy here and potentially some more performant ways to achieve this.
```

Comment:
Should these be prompts/questions instead of examples, to fit better with the rest of the template?